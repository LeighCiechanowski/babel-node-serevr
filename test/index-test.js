import request from 'supertest';
import server from '../src/index';

afterEach(() => {
  server.close();
});

describe('routes: index', () => {
  test('should respond as expected', async () => {
    const response = await request(server).get('/ping');
    expect(response.status).toEqual(200);
    expect(response.body.data).toEqual('pong');
  });
});
