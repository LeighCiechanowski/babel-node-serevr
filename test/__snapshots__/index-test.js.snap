// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes: index should respond as expected 1`] = `
Object {
  "data": "pong",
}
`;
